apis/kueue/v1alpha2/resourceflavor_types.go

```diff
@@ -29,14 +29,6 @@ type ResourceFlavor struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// labels associated with this flavor. They are matched against or
```

Comment:
we still need this comment on the API saying that those labels mirror the ones on the nodes the flavor targets.

Reply:
Right... moved up to the main object.

I tried putting it over metadata, but it's not included in the CRD yaml.